tools/docker/tests/test_flows.sh

```diff
@@ -21,9 +21,13 @@ error=0
 start_gw_repeater() {
     dbg "delete running containers before starting test"
     eval docker rm -f gateway $redirect
-    eval docker rm -f repeater $redirect
-    dbg "start gw and repeater"
-    eval ${scriptdir}/test_gw_repeater.sh -d 5 $redirect
+    eval docker rm -f repeater $redirect    
+    dbg "start gw and repeater"  
+    eval ${scriptdir}/test_gw_repeater.sh -f -d 5 $redirect|| 
```

Comment:
please add a space before `||`

Reply:
Yes, in my branch the start_gw_repeater deletes 3 containers since I have 2 repeaters.
maybe in the future, the start_gw_repeater method would be more necessary.